### ISSUE TITLE
Package not compatible with remotion 4.

### ISSUE THREAD

**Issue by LoganTann** [fwextensions/remotion-time#2]
This package requires Remotion 3. The following error appears when installing it with remotion 4. 

```
$ npm install remotion-time --save-dev

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: vda-2024@1.0.0
npm ERR! Found: remotion@4.0.12
npm ERR! node_modules/remotion
npm ERR!   remotion@"4.0.12" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer remotion@"^3.3.54" from remotion-time@0.0.2
npm ERR! node_modules/remotion-time
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

```

When forcing the installation, this package seems to be working on remotion 4.

**Comment by mohsentaleb** [comment]
@LoganTann Until the maintainer updates the peer dependency versions, you can use `npm install remotion-time --legacy-peer-deps`. It already works with remotion v4.

**Comment by LoganTann** [comment]
Thanks for the reply @mohsentaleb, this is what I did 👍

**Comment by fwextensions** [comment]
Fixed in 0.0.3.
